packages/asset-service/src/generateAssetData/generateAssetData.ts

```diff
@@ -20,6 +20,10 @@ const generateAssetData = async () => {
     `./src/service/generatedAssetData.json`,
     JSON.stringify(generatedAssetData)
   )
+
+  const assetService = new AssetService()
+
+  await assetService.initialize()
```

Comment:
I don't see `assetService` being called anywhere else.

Reply:
good call. was using this for testing. i can remove it